.nvmrc

```diff
@@ -0,0 +1 @@
+13.12
```

Comment:
I had to upgrade my node version to get `yarn install` to pass as per https://github.com/nodejs/node/issues/32852#issuecomment-629692323